scripts/output/comparison/compareScenarios2.R

```diff
@@ -116,10 +86,15 @@ profiles <- remind2::getCs2Profiles()
 
 # Let user choose cs2 profile(s).
 profileNamesDefault <- determineDefaultProfiles(outputdirs[1])
-profileNames <- chooseFromSequence(
-  names(profiles),
-  "Choose profiles for cs2.",
-  profileNamesDefault)
+profileNames <- gms::chooseFromList(
+  ifelse(names(profiles) %in% profileNamesDefault, crayon::cyan(names(profiles)), names(profiles)),
```

Comment:
With this code the resulting `profileNames` may contain the crayon markup. Despite that it works on the cluster, as  `profileNames` is used to create a slurm command that is then executed via `system()`. This seems to "clear" the crayon markup. But for local use it probably will crash. So we need to use `returnBoolean` here, I guess

Reply:
You are right, I did correct that.